### DISCUSSION TITLE
Phone verification with Twilio

### DISCUSSION THREAD

**Discussion by jbspears** [zitadel/zitadel#8612] [Q&A] [answered]
I haven't been able to find any information on how to verify a phone number using Twilio as SMS provider.  It will send a message with a code generated by Zitadel and I respond back with the code and get back a response from Twilio, but in Zitadel I always see "not verified".  Is there a webhook to add in Twilio?  Do we have to implement this ourselves?  FYI we are self-hosting and using the latest version (2.61.0).  Thanks!

**Comment by hifabienne** [comment, marked as answer]
In the zitadel management ui you can verify your phone number.
Or you can use our API to do so, depending on your use case.

your-domain/ui/console/users/me?id=general
<img width="800" alt="image" src="https://github.com/user-attachments/assets/fccd3031-c1ed-4f48-ad9f-27cf283f1458">

https://zitadel.com/docs/apis/resources/user_service_v2/user-service-verify-phone
